datalad_next/create_sibling_webdav.py

```diff
@@ -369,41 +371,100 @@ def _create_sibling_webdav(
     # simplify downstream logic, export yes or no
     export_storage = 'export' in storage_sibling
 
+    existing_siblings = [r[1] for r in
+                         _yield_ds_w_matching_siblings(ds,
+                                                       (name, storage_name),
+                                                       recursive=False)
+                         ]
+
+    results = []
+
     if storage_sibling != 'no':
-        yield from _create_storage_sibling(
-            ds,
-            url,
-            storage_name,
-            existing,
-            credential,
-            export=export_storage,
-        )
+        if storage_name in existing_siblings:
+            if existing == 'skip':
+                r = get_status_dict(ds=ds,
+                                    status='notneeded',
+                                    message=f"Skipped on existing "
+                                            f"sibling {storage_name}")
+            elif existing == 'reconfigure':
+                r = _create_storage_sibling(
+                        ds,
+                        url,
+                        storage_name,
+                        credential,
+                        export=export_storage,
+                        reconfigure=True,
+                    )
+            else:
+                # Shouldn't happen, since 'error' was treated upfront
+                raise ValueError(f"Unexpected value of 'existing': {existing}")
```

Comment:
Would be great to put this conditional into a helper fx:

```py
if storage_name in existing_siblings:
  r = dowhatisneededifthingsexist(...,existing)
```

Reply:
Done.